server/webapp/WEB-INF/rails/webpack/views/components/collapsible_panel/index.tsx

```diff
@@ -26,6 +26,7 @@ const classnames = bind(styles);
 type AttributeType = m.Children;
 
 export interface Attrs<Header, Actions> {
+  dataTestId?: string;
```

Comment:
@akshaydewan -- this needs to explicitly added, if we do `{...vnode.attrs}`, that is adding other `attrs` as well such as `header`, `actions`, `error` etc.